webapp/channels/src/actions/command.ts

```diff
@@ -59,7 +59,7 @@ export function executeCommand(message: string, args: CommandArgs): ActionFuncAs
         }
         const cmd = msg.substring(0, cmdLength).toLowerCase();
         if (cmd === '/code') {
-            msg = cmd + ' ' + msg.substring(cmdLength, msg.length).trimEnd();
+            msg = cmd + msg.substring(cmdLength, msg.length).trimEnd();
```

Comment:
can you walk me through your decision on removing that space? I think there are other options available but I'd like to peek into your decision why you chose this one.

Reply:
Firstly i started inspecting from code_block component. After that i backtracked the msg from message_html_to_component to post_markdown and post_list to advance text editor. I looked for all solutions that i can go with, I came to conclusion that this would be good to do was this is where i found where space was redundant, Like when you print the msg in this component now after this change it would be /code msg before that it was /code  msg. I also looked at the backend call done here which was /execute and its working as it should be. So i thought this was the best solution to remove the space from here.

Reply:
awesome, but have you tried changing trimEnd to just trim? if that works, then I think we can make the whole if unnecessary, making the file cleaner. Don't you think?

Reply:
Yeah, its an great idea which will make the file cleaner. I have made the change.